source/docs/hardware-components/fasteners-guide.rst

```diff
@@ -0,0 +1,182 @@
+   :width: 150px
+
+These bolts have a cylindrical head and are tightened and loosened with a hexagonal recess.
+
+**Button Head Bolts**
+
+.. figure:: images/botton-head.jpeg
+   :alt: diagram showing thread terminology
+   :width: 150px
+
+These bolts have a domed head, in contrast to the flat top of a standard socket bolt.
+
+**Grub Screws**
+
+.. figure:: images/grub-screw.jpg
+   :alt: diagram showing thread terminology
+   :width: 150px
+
+These bolts have no head, the recess is instead embedded within the threads. They are primarily used as set screws.
+
+**Hex Bolts**
+
+.. figure:: images/hex-head.jpg
+   :alt: diagram showing thread terminology
+   :width: 150px
+
+
+The bolts are tightened with an external hex-shaped head. When designing for these bolts, it is important to allow clearance on the side of the head for tightening.
+
+
+Every bolt requires a "drive" -- meaning the recess which is used to turn and install it. Each of the bolts listed above can have a variety of different types of drives:
+
+.. figure:: images/types-of-drives.png
+   :alt: diagram showing thread terminology
+
+Nuts
+^^^^
+
+A nut is a fastener with an internally threaded hole. Together with a bolt, they're used to join parts together.
+
+Here are some types of nuts you may encounter in FTC:
+
+
+**Hex Nuts**
+
+.. figure:: images/hex-nut.jpg
+   :alt: diagram showing thread terminology
+   :width: 150px
+
+These most common type of nut. They can be tightened with a socket hex wrench
+
+**Square Nuts**
+
+.. figure:: images/square-nut.jpg
+   :alt: diagram showing thread terminology
+   :width: 150px
+
+These are four-sided nuts.
+
+**T-slot Nuts**
+
+.. figure:: images/t-slot-nut.jpg
+   :alt: diagram showing thread terminology
+   :width: 150px
+
+These can be inserted into t-slot extrusion to tighten parts onto it.
+
+**Wing Nuts**
+
+.. figure:: images/wing-nut.jpg
+   :alt: diagram showing thread terminology
+   :width: 150px
+
+These nuts can be easily tightened without a wrench.
+
+**Nyloc Nuts**
+
+.. figure:: images/nyloc-nut.jpg
+   :alt: diagram showing thread terminology
+   :width: 150px
+
+These nuts contain a nylon ring which prevents loosening. We recommend using these wherever possible.
+
+Making Threads
+^^^^^^^^^^^^^^
+
+Sometimes, you will want to create new threads. This process is called threading.
+To create internal threads, you can use a cutting tool called a tap -- essentially a sharper version of a screw.
+
+To create external threads, you use a tool called a die.
+
+.. figure:: images/tap-and-die.jpeg
+   :alt: diagram showing thread terminology
+   :width: 250px
+
+
+Threadlocker
+^^^^^^^^^^^^
+
+Bolts on your robot may loosen over time, especially if there are heavy vibrations. To prevent this you may use locking nuts or lock washers, which mechanically press against the thread. Another solution is to use a threadlocker, which is an adhesive compound that -- as it's name suggests -- locks threads in place.
```

Comment:
I would recommend moving the content from [this section](https://gm0.org/en/latest/docs/common-mechanisms/power-transmission/power-transmission-glossary.html#term-Loctite) over here, as well as using the image from there as well.